src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days >= 7) {
+    result = days * 40 - 50;
```

Comment:
```suggestion
    return days * 40 - 50;
```
the result variable is redundant, you may return the calculated value immediately